### DISCUSSION TITLE
HFS on a Raspbarry PI

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#521] [General]
Hi,

I wonder if HFS will run on a Raspbarry Pi. I have not much knowledge about the Raspberry Pi, but it runs on some sort of Linux (Debian)? It would be nice if it did, because then I am thinking to move my shared data to an external Harddrive connected to a Raspberry Pi.

So ... Just wondering if it is possible...

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
I'm quite sure it will work

**Reply by PH5HP** [reply]
Hi,

Nice ... Any idea what Pi at least I have to use?... 3b Ok?...

Kind regards,
Henk - https://fileserver.ph5hp.nl/

**Comment by rejetto** [comment]
Quite sure any will do

**Reply by PH5HP** [reply]
Thank you ...
